tests/core/test_db.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import pytest_asyncio
+
+
+@pytest_asyncio.fixture
+async def json_test_table(db):
+    try:
+        await db.execute("CREATE TABLE test (data JSON)")
+        yield db
+    finally:
+        await db.execute("DROP TABLE test")
+
+
+@pytest.mark.asyncio
+async def test_sql_json(db, json_test_table):
+    obj = {"a": "foo", "b": "bar"}
```

Comment:
- can you please add a nested field and query for it please

```suggestion
    obj = {"a": "foo", "b": "bar", "c": {"x": "zzz"}}
```

- are queries for numbers possible (using `>`, `<=`, etc)?

Reply:
numbers are a bit tricky. Postgres makes it a bit difficult here as we would need to explicitly cast the accessed object into a number for such comparison

Reply:
one solution for this is to add functionality to json_path to specify a type which is needed for postgresqls stronger type system which is no problem for sqlite